addons/website_crm_partner_assign/data/res_partner_activation_data.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="res_partner_activation_fully_operational" model="res.partner.activation">
```

Comment:
Naming: better keep the 'data' naming we generally use to indicate this is a master data, like ``res_partner_activation_data_fully_operational`` (same for other data)